README.md

```diff
@@ -286,6 +286,7 @@ The `vue-eslint-parser` uses the parser which is set by `parserOptions.parser` t
 2. Make sure your tool is set to lint `.vue` files.
   - CLI targets only `.js` files by default. You have to specify additional extensions by `--ext` option or glob patterns. E.g. `eslint "src/**/*.{js,vue}"` or `eslint src --ext .vue`.
   - VSCode targets only JavaScript or HTML files by default. You have to add `"vue"` to the `"eslint.validate"` array in vscode settings. e.g. `"eslint.validate": [ "javascript", "javascriptreact", "vue" ]`
+  - Set VSCode Vetur settings: `"vetur.validation.template": false` to avoid Vetur template validation overriding plugin validation.
```

Comment:
I think we should just point to this resource: https://github.com/vuejs/vetur/blob/master/docs/linting-error.md
This contains nice instruction of how to best use and configure vetur to work nicely with eslint-plugin-vue

Reply:
Yea. So we could write something along "If you use Vetur, see [here](https://github.com/vuejs/vetur/blob/master/docs/linting-error.md) how to configure `eslint-plugin-vue` for Vetur."

The bottom line, if you use Vetur, which supplies their own default configuration for this plugin, you have to disable their template validation completely and supply your custom configuration, preferably based on theirs to keep it consistent, alternatively based on what is suggested in our readme.